packages/orbit-components/src/Tooltip/Tooltip.stories.js

```diff
@@ -93,6 +93,21 @@ storiesOf("Tooltip", module)
         "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
     },
   )
+  .add(
+    "Full-width",
+    () => {
+      const content = text("content", "Write your text here.");
+      return (
+        <Tooltip fullWidth content={content}>
+          <Button fullWidth>Full-width</Button>
+        </Tooltip>
+      );
+    },
+    {
+      info:
+        "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
```

Comment:
This description is related to the `playground` story, not to the current one.

Reply:
I just copy-pasted the info from other Tooltip stories, but I'll remove this because it doesn't make sense.